api/src/main/java/org/openmrs/OrderSetMember.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.io.Serializable;
+
+/**
+ * @since 1.12
+ * OrderSetMember
+ */
+public class OrderSetMember extends BaseOpenmrsMetadata implements Serializable {
+	
+	public static final long serialVersionUID = 72232L;
+	
+	private Integer orderSetMemberId;
+	
+	private OrderType orderType;
+	
+	private String orderTemplate;
+	
+	private Integer sortWeight;
+	
+	private Concept concept;
+
```

Comment:
I have added a back link. But as a part of the implementation. This was not needed, to the API to set the orderSet in an orderSetMember is not there currently